tests/components/holiday/test_config_flow.py

```diff
@@ -0,0 +1,68 @@
+"""Test the Holiday config flow."""
```

Comment:
We need to test aborts as well when the config entry already exists, both country and country+province.

Reply:
Done.